editors/MacVim/Portfile

```diff
@@ -25,9 +25,9 @@ long_description \
 
 homepage            https://macvim-dev.github.io/macvim/
```

Comment:
Should update the homepage to https://macvim.org/ per release notes.

Reply:
Fixed in thomasnyman@5bca4e078d27901f63dfaba150e134d1f99b71be.